### PR TITLE
Slides for April 2017

### PR DESCRIPTION
**Pull request by lcobucci**
Also updated the raffler slides in the end so that @rdohms don't raffle and then asks for people to check-in 😂 

Preview: https://lcobucci.github.io/monthly-slidedeck/2017-April.html

### CODE REVIEW

File: 2017-April.html [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+                <section>
+                    <h1 class="slide fancy">PHPKonf 2017</h1>
+                    <img style="" height="170" src="img/conference-logos/phpkonf.png">
+                    <h4>May 20th</h4>
+                    <h4>Istanbul</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Srbija Conference 2017</h1>
```
Comment (pascaldevink) [inline 2017-April.html]: Tickets are sold out, so don't think it's worth mentioning

File: 2017-April.html [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+                        We're hiring!
+                    </div>
+                </section>
+
+                <section>
+                    <h1>Shirts!</h1>
+
+                    <img src="img/projects/gitkraken.svg" alt="Sponsor: GitKraken" style="height: 300px;">
+
+                    <a class="fancy" href="https://support.gitkraken.com/#get-started">https://support.gitkraken.com/#get-started</a>
```
Comment (lcobucci) [inline 2017-April.html]: Sure, forgot it

Comment (lcobucci) [inline 2017-April.html]: Updated 😉

**Comment by rdohms** [comment]
Your commit message does not state why these slides were created and does not make reference to tickets. We have standards you know?

**Comment by WyriHaximus** [comment]
Go home @rdohms you're drunk

**Comment by rdohms** [comment]
@WyriHaximus you just don't work with @lcobucci :D

**Comment by WyriHaximus** [comment]
@rdohms hey you hired him 😜

**Pull request merged**